provision/acc_provision/apic_provision.py

```diff
@@ -3456,5 +3412,113 @@ def cloudfoundry_tn(self):
         return path, data
 
 
+# Open port 2379 only for openshift
+def open_port_for_openshift(data):
+
+    os_extra_port = collections.OrderedDict(
+        [
+            (
+                "vzEntry",
+                collections.OrderedDict(
+                    [
+                        (
+                            "attributes",
+                            collections.OrderedDict(
+                                [
+                                    (
+                                        "name",
+                                        "openshift-svc-catalog",
+                                    ),
+                                    (
+                                        "etherT",
+                                        "ip",
+                                    ),
+                                    (
+                                        "prot",
+                                        "tcp",
+                                    ),
+                                    (
+                                        "dFromPort",
+                                        "2379",
+                                    ),
+                                    (
+                                        "dToPort",
+                                        "2379",
+                                    ),
+                                    (
+                                        "stateful",
+                                        "no",
+                                    ),
+                                    (
+                                        "tcpRules",
+                                        "",
+                                    ),
+                                ]
+                            ),
+                        )
+                    ]
+                ),
+            )
+        ]
+    )
+
+    data['fvTenant']['children'][7]['vzFilter']['children'].append(os_extra_port)
+
+
+def open_port_for_dockerucp(data):
+
+    single_ports = ["443", "12376", "10250", "12378", "12379", "12380", "12381",
```

Comment:
Can we split this into single ports and ranges?

Reply:
Done